### PR TITLE
91 emergency repair identification priority list page options

### PR DESCRIPTION
**Pull request by elena-vi**
In this PR:
- [x] Reorganised the user journey to reflect [beta prototype V2](https://southwark-repairs-alpha-master.herokuapp.com/beta/v2/)
- [x] Add smell gas page
- [x] Add communal page
- [x] Add more options to priority list
- [x] Handle the user pressing the back button
- [x] Add cypress tests for the relevant journeys 
- [ ] After going back, the forwards browser button does not work

### CODE REVIEW

**Comment by Bhavesh-Patel** [comment]
### Issue
After using browser back navigation to change previous choice, when new option is selected, emergency information is not shown.

### Steps to reproduce

1. Select gas or an emergency option and click continue.
2. Use browser back navigation.
3. Again select gas or an emergency option and click continue.

### Expected
Should show respective emergency information.

### Actual
Shows communal area question.

### Workaround
If back link clicked after step 2 (above), no navigation takes place and when new choice made, correct emergency information is shown.

**Comment by Bhavesh-Patel** [comment]
### Issue
On the [emergency identification page](https://purple-desert-05060ea03-9.westeurope.azurestaticapps.net/report-repair/priority-list), clicking the back link navigates to the [repair location page](https://purple-desert-05060ea03-9.westeurope.azurestaticapps.net/report-repair/address).
Clicking the back link again, shows the [repair location with postcode input page](https://purple-desert-05060ea03-9.westeurope.azurestaticapps.net/report-repair/postcode)
A further click, shows the [resident type page](https://purple-desert-05060ea03-9.westeurope.azurestaticapps.net/report-repair/resident-type) (currently blank).
A subsequent click, shows the [landing page](https://purple-desert-05060ea03-9.westeurope.azurestaticapps.net/).

### Steps to reproduce

1. Begin reporting repair
2. Click back link on emergency identification page.
3. Continue clicking back link until landing page is show.

### Expected
Should show [landing page](https://purple-desert-05060ea03-9.westeurope.azurestaticapps.net/).

### Actual
Shows [repair location with address input page](https://purple-desert-05060ea03-9.westeurope.azurestaticapps.net/report-repair/address).

File: tests/cypress/integration/reportRepair/priorityList.spec.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      cy.get('button').click();
+      cy.go('back')
+      cy.contains('I have no heating').click();
+      cy.get('button').click()
+      cy.url().should('include', '/report-repair/emergency-repair');
+    })
+  })
+
+  context('User uses back buttons to navigate out of an exit page and selects the same option', ()=>{
+    it('should redirect the user to a different exit page',  () => {
```
Comment (Bhavesh-Patel) [inline tests/cypress/integration/reportRepair/priorityList.spec.js:96]: Shouldn't the `it` be about redirecting to the same page?

File: tests/cypress/integration/reportRepair/priorityList.spec.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+  context('When a user selects: Something else', ()=>{
+    it('should redirect them to communal page',  () => {
+      cy.contains('Something else').click();
+      cy.get('button').click()
+      cy.url().should('include', '/report-repair/communal');
+    });
+  })
+
+  context('User uses back buttons to navigate out of an exit page and selects a different option', ()=>{
```
Comment (Bhavesh-Patel) [inline tests/cypress/integration/reportRepair/priorityList.spec.js:84]: Should this test description include that the different option is also an emergency option?

File: src/flow.js [hunk 3]
```diff
@@ -108,13 +111,18 @@ export default class Flow {
     // workout what the new previous step should be.
     if (this._prevStepIsNotDefinedOrEqualsCurrentStep(state)) {
       if (this._prevStepIsInFlow(state)) {
+        console.log(state)
```
Comment (Bhavesh-Patel) [inline src/flow.js:114]: Unless they are necessary, remove `console.log` from code - there are a few of them between lines 114 and 125.

**Comment by elena-vi** [comment]
[Emergency repair identification (priority list page + options)](https://trello.com/c/31kbgjgx/91-emergency-repair-identification-priority-list-page-options)

**Pull request merged**